krun/scheduler.py

```diff
@@ -112,7 +125,7 @@ def _parse(self):
             assert False
 
         # Check we saw all the necessary header fields
-        assert seen_headers == ManifestManager.HEADER_FIELDS
+        assert ManifestManager.HEADER_FIELDS.issubset(seen_headers)
```

Comment:
OK, but I'm not sure this is a very strong assertion. Perhaps another way would be to have the per-platform fields stored in a set inside each `Platform` class, then you could check whether the seen headers were exactly correct.